src/Networking/MultiplayerManager.cs

```diff
@@ -191,6 +192,19 @@ private static void QueueMainThread(Action action)
             }
         }
 
+        /// <summary>
+        ///     Get the Player object by username. Warning, expensive call!!!
+        /// </summary>
+        public static Player GetPlayerByUsername(string username)
```

Comment:
Maybe move this method to the Server class, because it only works when the game acts as a server.